### PR TITLE
Add Users count endpoint

### PR DESCRIPTION
**Pull request by trick0**
#### Description of what you did:
Added users count

### CODE REVIEW

File: packages/strapi-plugin-users-permissions/controllers/User.js [hunk 1]
```diff
@@ -52,6 +52,18 @@ module.exports = {
...
   },
 
+  /**
+   * Retrieve user count.
+   * @return {Number}
+   */
+
+  async count(ctx) {
+    if (ctx.query._q) {
+      return await strapi.plugins['users-permissions'].services.user.countSearch(ctx.query);
```
Comment (alexandrebodin) [inline packages/strapi-plugin-users-permissions/controllers/User.js:62]: You didn't implement this method in the service :)

Comment (trick0) [reply packages/strapi-plugin-users-permissions/controllers/User.js:62]: I'm sorry, totally forgot to add it!

**Comment by alexandrebodin** [comment]
@trick0 Thanks for the quick update :); Can you verify the DCO check in the PR checks so the PR is mergeable :)

**Comment by alexandrebodin** [comment]
Hi @trick0 sorry but the DCO check is still not passing :/

**Comment by trick0** [comment]
Hi @alexandrebodin, sorry but I don't really know how to solve this.
I'm following what has been said [here](https://github.com/strapi/strapi/pull/5859/checks?check_run_id=621621658) but nothing seems working. Could you guide me to the solution?
Sorry :/

**Comment by alexandrebodin** [comment]
Hi, I cannot really help you here :) You should rebase interactively with the signoff option and drop the merge commit you made so the history is cleaned up

**Comment by alexandrebodin** [comment]
Hi @trick0  any update on your PR ?

**Comment by baermathias** [comment]
@derrickmehaffy Is there anything in particular that needs to be considered when calling `/users/count` ?
I have 3 users and the endpoint is activated for authenticated role. When calling it I get `0` as response.

**Pull request merged**